### PR TITLE
[docs] add changelog for 0.16.1

### PR DESCRIPTION
**Pull request by miherlosev**
@DevExpress/testcafe-docs

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v0.16.1 (2017-6-19)
+
+### Bug Fixes
+
+* We need to process a markup imported via meta[rel="import"] html ([testcafe-hammerhead/#1161](https://github.com/DevExpress/testcafe-hammerhead/issues/1161))
```
Comment (kirovboris) [inline CHANGELOG.md]: `meta[rel="import"]`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+* Wrong passing context to a MutaionObserver ([testcafe-hammerhead/#1178](https://github.com/DevExpress/testcafe-hammerhead/issues/1178))
+* We don't need to process an 'innerHTML' property if element doesn't have it ([testcafe-hammerhead/#1164](https://github.com/DevExpress/testcafe-hammerhead/issues/1164))
```
Comment (kirovboris) [inline CHANGELOG.md]: 'innerHTML' -> `innerHTML`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v0.16.1 (2017-6-19)
+
+### Bug Fixes
+
+* We need to process a markup imported via meta[rel="import"] html ([testcafe-hammerhead/#1161](https://github.com/DevExpress/testcafe-hammerhead/issues/1161))
+* Wrong passing context to a MutaionObserver ([testcafe-hammerhead/#1178](https://github.com/DevExpress/testcafe-hammerhead/issues/1178))
```
Comment (kirovboris) [inline CHANGELOG.md]: `MutaionObserver`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v0.16.1 (2017-6-19)
+
+### Bug Fixes
+
+* We need to process a markup imported via `meta[rel="import"]` ([testcafe-hammerhead/#1161](https://github.com/DevExpress/testcafe-hammerhead/issues/1161))
```
Comment (inikulin) [inline CHANGELOG.md]: We need to rephrase all entries here

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,13 @@
...
+* Wrong passing context to a `MutationObserver` ([testcafe-hammerhead/#1178](https://github.com/DevExpress/testcafe-hammerhead/issues/1178))
+* We don't need to process an `innerHTML` property if element doesn't have it ([testcafe-hammerhead/#1164](https://github.com/DevExpress/testcafe-hammerhead/issues/1164))
```
Comment (VasilyStrelyaev) [inline CHANGELOG.md]: * Markup imported via meta[rel="import"] is now processed. ([testcafe-hammerhead/#1161](https://github.com/DevExpress/testcafe-hammerhead/issues/1161))
* The correct context is passed to `MutationObserver`. ([testcafe-hammerhead/#1178](https://github.com/DevExpress/testcafe-hammerhead/issues/1178))
* The `innerHtml` property is no longer processed for elements that don't have this property.  ([testcafe-hammerhead/#1164](https://github.com/DevExpress/testcafe-hammerhead/issues/1164))

**Comment by VasilyStrelyaev** [comment]
@miherlosev We need to add two more items to the changelog. Place them at the beginning, before hammerhead bugs.

```
* A typo in RoleOptions typedefs was fixed ([#1541](https://github.com/DevExpress/testcafe/issues/1541))
* TestCafe no longer crashes on node 4 with an unmet dependency ([#1547](https://github.com/DevExpress/testcafe/issues/1547))
```

**Comment by miherlosev** [comment]
> We need to add two more items to the changelog. 

Done

**Pull request merged**